curriculum/challenges/english/01-responsive-web-design/css-piano/part-005.md

```diff
@@ -0,0 +1,64 @@
+---
+id: 612e783368153d2061d57c05
+title: Part 5
+challengeType: 0
+dashedName: part-5
+---
+
+# --description--
+
+You also need to specify the character encoding for the HTML. Add a `meta` tag above your `title` element. On that new tag, set the `charset` attribute to `UTF-8`.
+
+# --hints--
+
+You should add a new `meta` element.
+
+```js
+const metas = document.querySelectorAll('meta');
+assert(metas?.length === 2);
+```
+
+Your new `meta` element should come before your `title` element.
```

Comment:
Whilst most common, and there is a reason for the placement of the `meta`, I worry we will get feedback like:
> I put my `meta` in a perfectly acceptable place (below the title), but it fails the test...